sdk/search/Azure.Search.Documents/src/SearchDocument/SearchDocument.cs

```diff
@@ -156,7 +154,6 @@ public partial class SearchDocument
         /// <returns>The value of the property.</returns>
         public IReadOnlyList<string> GetStringCollection(string key) => GetValue<string[]>(key);
 
-#if EXPERIMENTAL_SPATIAL
```

Comment:
This is a public API change. I want to confirm we want these 2 methods.

Reply:
We don't normally refer to "Collection" in member names. Maybe just `GetPoints`?

/cc @KrzysztofCwalina @tg-msft

Reply:
Collection properties in `SearchDocument` are named as xxxCollection.
Some other examples - `GetBooleanCollection`, `GetDoubleCollection`.